<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instructor</class>
 <widget class="QDialog" name="instructor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(223, 228, 215);</string>
       </property>
       <layout class="QVBoxLayout" name="tab">
        <item>
         <widget class="QPushButton" name="aboutMeButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>About me</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="courseDetailsButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>Courses details</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerStudentsButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>Register students</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="logOutButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>Log out</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="emptyPage">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_19">
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_21">
                <property name="styleSheet">
                 <string notr="true">
font: 23pt &quot;Ubuntu Mono&quot;;</string>
                </property>
                <property name="text">
                 <string>Welcome to </string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_25">
                <property name="styleSheet">
                 <string notr="true">
font: 23pt &quot;Ubuntu Mono&quot;;</string>
                </property>
                <property name="text">
                 <string>instructor view</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="aboutMePage">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="addCourseWindow">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;</string>
                 </property>
                 <property name="text">
                  <string>My information</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="styleSheet">
                      <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                     </property>
                     <property name="text">
                      <string>ID</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="idField">
                     <property name="text">
                      <string>[ID]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="styleSheet">
                      <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                     </property>
                     <property name="text">
                      <string>Name</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="nameField">
                     <property name="text">
                      <string>[Name]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="styleSheet">
                      <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                     </property>
                     <property name="text">
                      <string>Department</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="departmentField">
                     <property name="text">
                      <string>[Department]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="styleSheet">
                      <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                     </property>
                     <property name="text">
                      <string>Salary</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="salaryField">
                     <property name="text">
                      <string>[Salary]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item alignment="Qt::AlignHCenter">
                   <widget class="QLabel" name="label_7">
                    <property name="styleSheet">
                     <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Update my info</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <widget class="QLabel" name="label_6">
                        <property name="styleSheet">
                         <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                        </property>
                        <property name="text">
                         <string>Name</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="inputNameField"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_39">
                      <item>
                       <widget class="QLabel" name="label_8">
                        <property name="styleSheet">
                         <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                        </property>
                        <property name="text">
                         <string>New password</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="inputNewPasswordField">
                        <property name="echoMode">
                         <enum>QLineEdit::Password</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="updateButton">
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="coursesDetailsPage">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="idTestField">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;</string>
                </property>
                <property name="text">
                 <string>Courses assigned details</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_20">
                  <item>
                   <widget class="QPushButton" name="fallButton">
                    <property name="text">
                     <string>Fall</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="springButton">
                    <property name="text">
                     <string>Spring</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="selectionStatusText">
                  <property name="styleSheet">
                   <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                  </property>
                  <property name="text">
                   <string>[Selection status]</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Year</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="yearComboBox"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Select courseID assigned</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="coursesComboBox"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Course information</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_10">
                      <item>
                       <widget class="QLabel" name="label_16">
                        <property name="text">
                         <string>Title</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="titleField">
                        <property name="text">
                         <string>[Title]</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_11">
                      <item>
                       <widget class="QLabel" name="label_17">
                        <property name="text">
                         <string>Department</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="departmentField_2">
                        <property name="text">
                         <string>[Department]</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_43">
                      <item>
                       <widget class="QLabel" name="label_18">
                        <property name="text">
                         <string>Credits</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="creditsField">
                        <property name="text">
                         <string>[Credits]</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_11">
                    <item>
                     <widget class="QLabel" name="label_56">
                      <property name="styleSheet">
                       <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                      </property>
                      <property name="text">
                       <string>Students registered to the course</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QTableView" name="studentsRegistered"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="registerStudentsPage">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="label_57">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;</string>
                </property>
                <property name="text">
                 <string>Register students to the courses</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_14">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Course</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="coursesComboBox2"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_14">
                  <item>
                   <widget class="QLabel" name="label_15">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>year</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="yearComboBox2"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_15">
                  <item>
                   <widget class="QLabel" name="label_19">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>semester</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="semesterComboBox2"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_16">
                  <item>
                   <widget class="QLabel" name="label_20">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>StudentID</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="studentIdComboBox"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_17">
                  <item>
                   <widget class="QLabel" name="label_22">
                    <property name="styleSheet">
                     <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Student name</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="stdName">
                    <property name="text">
                     <string>[Name and surname]</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_24">
                  <property name="styleSheet">
                   <string notr="true">font: 700 14pt &quot;Ubuntu Condensed&quot;;</string>
                  </property>
                  <property name="text">
                   <string>SecID</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="secIdComboBox"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="registerButton">
                <property name="text">
                 <string>Register</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
